### DISCUSSION TITLE
Java 11 (LTS) and Java 17 (LTS)

### DISCUSSION THREAD

**Discussion by sreev** [nextgenhealthcare/connect#4559] [Feature/Enhancement Ideas]
Is your feature request related to a problem? Please describe.
Current java versions supported are java 8 and java 9.
Upgrade to Java 11 (LTS) and Java 17 (LTS).

https://en.wikipedia.org/wiki/Java_version_history

Describe your use case
Java 8 and 9 are in extended support period.
Java 11 (LTS) is the prevailing current version in use.

Describe the solution you'd like
Java 17 (LTS) is going to be out in Sep 21. It is right about the time, to start looking into adopting it.
`Also, take this opportunity to define how to align Connect releases with Java releases, from now on.`

Describe alternatives you've considered
None.

Additional context
None.

**Comment by tonygermano** [comment]
Current versions of Mirth support all java versions 8+. Support does not stop at Java 9.
